Automattic/wp-calypso
# Fix VideoPress Variant Selector

#### Changes proposed in this Pull Request

* Use a common `type` for both VideoPress products

#### Testing instructions

1. Add VideoPress to cart
2. Verify that in staging/production ( either will work ), the variants of VideoPress are not shown in the Cart
3. Verify that on the branch, the variants are shown

| Staging | This Branch |
| ------- | ----------- |
| <img width="562" alt="Screen Shot 2022-01-20 at 09 26 58" src="https://user-images.githubusercontent.com/2810519/150391063-e02c2b49-9bab-407c-b048-96d03a6da72f.png"> |<img width="565" alt="Screen Shot 2022-01-20 at 09 26 34" src="https://user-images.githubusercontent.com/2810519/150391127-5ebceec9-09b8-474e-b87a-c3110afb2a19.png"> |
